types/site/_collections/tags.d.ts

```diff
@@ -26,6 +26,7 @@ declare global {
     posts: {
       [locale: string]: EleventyCollectionItem[];
     };
+    url: string;
```

Comment:
I saw this above; why can't we use the peramalink somehow here?

Reply:
Permalinks are generated based on pagination, this is kinda like a source of truth.